src/components/catigories.js

```diff
@@ -1,7 +1,11 @@
+import { useDispatch } from 'react-redux';
+import { underConstruction } from '../redux/categories/categories';
+
 function Categories() {
+  const dispatch = useDispatch();
   return (
     <>
-      <button type="button">Check Status</button>
+      <button type="button" onClick={() => dispatch(underConstruction)}>Check Status</button>
```

Comment:
- [ ] [OPTIONAL] Good job using dispatch on button click how ever you should display `Under construction in the page` not just in the store

Reply:
Done 👍🏻